tests/testthat/test-distributions.R

```diff
@@ -23,15 +23,16 @@ test_that("rmvnorm has correct behaviour in trivial case", {
 test_that("rvnorm produces correct expectation", {
   vcv <- matrix(c(4, 2, 2, 3), ncol = 2)
   f <- make_rmvnorm(vcv)
+  rng <- mcstate_rng$new(seed = 42)
   x <- c(1, 2)
-  set.seed(1)
-  r <- t(replicate(1e5, f(x)))
+  r <- t(replicate(1e5, f(x, rng)))
   expect_equal(colMeans(r), c(1, 2), tolerance = 0.01)
   expect_equal(var(r), vcv, tolerance = 0.01)
 })
 
 
 test_that("can compute the log density of a multivariate normal", {
+  skip_if_not_installed("mvtnorm")
```

Comment:
What's the idea with these skips? If we're running tests either manually or on CI, don't we want to run all of them, and install what's needed?

Reply:
Long story.  

* The package is only a "suggested" dependency (it's only used in tests) so this means that the test won't run if it's not available, which is what CRAN requires
* By default, the CI checks won't even start (the build will error almost immediately) if suggested packages are not found so we don't ever really see this.  However, it will be nice on (say) a train if the package is not there, which is what prompted me to remember to add this
* More annoyingly, CRAN will sometimes run checks with suggested packages missing, in order to send emails to people requiring that they apply this sort of fix, in order to complain they are overburdened.

Reply:
oh CRAN... 😆